example-frontend/components/Layout.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC, ReactNode } from "react";
+import Footer from "./Footer";
+import Header from "./Header";
+
+interface Props {
+    children: ReactNode
+}
+const Layout: FC<Props> = ({children}) => {
+    return (
```

Comment:
Scusi, but something is still not working with the layout.
![Xnapper-2023-10-10-14 01 22](https://github.com/ToucanProtocol/OffsetHelper/assets/9334296/503b36e4-cb15-4ab2-9e72-62c21074e775)